onedayhero-domain/src/main/java/com/sixheroes/onedayherodomain/mission/Mission.java

```diff
@@ -84,4 +96,21 @@ private void validateBookmarkCountAddable(MissionStatus missionStatus) {
     public void validRangeOfMissionTime(LocalDateTime dateTime) {
         missionInfo.validMissionDateTimeInRange(dateTime);
     }
+
+    private void validMissionOwner(
+        Long userId
+    ) {
+        if (!citizenId.equals(userId)) {
+            log.debug("미션 소유자가 아닙니다. userId : {}, citizenId : {}", userId, citizenId);
+            throw new IllegalArgumentException(ErrorCode.EM_007.name());
```

Comment:
```suggestion
        if (!Objects.equals(citizenId, userId)) {
            log.debug("미션 소유자가 아닙니다. userId : {}, citizenId : {}", userId, citizenId);
            throw new IllegalArgumentException(ErrorCode.EM_007.name());
```

개인적으로 npe에 방어적인 코드로 작성하는걸 선호합니다ㅎㅎ